terraso_backend/apps/core/models/users.py

```diff
@@ -47,6 +47,7 @@ class User(AbstractUser):
 
     username = None
     email = models.EmailField(unique=True)
+    profile_image_path = models.CharField(max_length=128, null=True)
```

Comment:
We should use a `URLField` here among other things, it's going to have a more reasonable length, for example.

Reply:
@cacarrara It will not be the complete URL, see: https://github.com/techmatters/terraso-backend/pull/74#discussion_r782353799

Reply:
@cacarrara Let me know if you have a suggestion for the length of the path, 128 felt reasonable, but let me know